### PR TITLE
Fixes 1188, 1190: ProfileChooser-FirstTime - Tablet (Landscape & Portrait) (Lowfi)

### LINKED ISSUE #1188: ProfileChooser-FirstTime - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/33741ea7-89da-4e5d-b2a4-9872f6db48b4/L-Profile-Chooser-First-Time

Implement low-fi UI for **ProfileChooser-FirstTime** tablet landscape mode

**Target PR date**: 3 June 2020
**Target completion date**: 6 June 2020

### LINKED ISSUE #1190: ProfileChooser-FirstTime - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/d63eabcd-f0bf-4e06-9c08-f4d77429c356/P-Profile-Chooser-First-Time 

Implement low-fi UI for **ProfileChooser-FirstTime** tablet portrait mode

**Target PR date**: 3 June 2020
**Target completion date**: 6 June 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1188 
- Fixes #1190 
- Implemented low-fi UI for both landscape and portrait mode in **ProfileChooser-FirstTime** tablet
- Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/33741ea7-89da-4e5d-b2a4-9872f6db48b4/L-Profile-Chooser-First-Time

## Notes
- I had to use `closeSoftKeyboard()` in two test cases because the `BackButton` closes the soft keyboard instead of performing what it is expected to perform, so I had to close the soft keyboard first then perform `pressBack()`
- I had to add a check for the type of the device _(Tablet or normal device)_ because the behavior is different in each case.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/values-sw600dp/booleans.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="isTablet">true</bool>
+</resources>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp/booleans.xml]: EOF is missing, add one blank line at the end.

File: app/src/main/res/values/booleans.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="isTablet">false</bool>
+</resources>
```
Comment (rt4914) [inline app/src/main/res/values/booleans.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/main/res/values/booleans.xml (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values/booleans.xml]: Done

File: app/src/main/res/values-sw600dp/booleans.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values-sw600dp/booleans.xml]: Done

**Comment by rt4914** [comment]
@MohamedMedhat1998 Delete this branch if not needed.

**Pull request merged**